### PR TITLE
Fixed code release

### PR DESCRIPTION
**Pull request by cyberomin**
Fixing release and adding github pipleine.

### CODE REVIEW

**Review by sircatalyst — APPROVED** [review]

File: modules/klump/klump.php [hunk 1]
```diff
@@ -23,7 +23,7 @@ public function __construct()
     {
         $this->name = 'klump';
         $this->tab = 'payments_gateways';
-        $this->version = '1.0.0';
+        $this->version = '0.1.0';
```
Comment (richienabuk) [inline modules/klump/klump.php:26]: Why not 1.0.0?
@cyberomin

Comment (cyberomin) [reply modules/klump/klump.php:26]: @richienabuk What does the SEMVER version say about versioning? There's still clean up that will happen.

**Review by 204070 — APPROVED** [review]

File: modules/klump/klump.php (hunk 1 above)
Comment (richienabuk) [reply modules/klump/klump.php:26]: Yeah but it should be 1.1.0 not 0.1.0.
@cyberomin

Comment (cyberomin) [reply modules/klump/klump.php:26]: @richienabuk ```
Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.``` Also, there is a bug in this code that isn't fixed yet.

**Review by 204070 — APPROVED** [review]

**Pull request merged**